geoapps/utils/statistics.py

```diff
@@ -16,18 +16,10 @@ def random_sampling(
     Perform a random sampling of the rows of the input array based on
     the distribution of the columns values.
 
-    Parameters
-    ----------
+    :param values: Input array of values N x M, where N >> M
+    :param size: Number of indices (rows) to be extracted from the original array
```

Comment:
preserve type in docstring, or, better, have it as type hint on the signature
```
def random_sampling(
    values: npt.NDArray[np.float64], size: int, method="histogram", n_bins=100, bandwidth=0.2, rtol=1e-4
) -> npt.NDArray[np.int_]:
```

See typing annotations for numpy: https://numpy.org/devdocs/reference/typing.html#api